roll/testing.py

```diff
@@ -81,6 +81,8 @@ class Client:
     # Default content type for request body encoding, change it to your own
     # taste if needed.
     content_type = 'application/json; charset=utf-8'
+    # Default headers to use eg. for patching Auth in tests.
+    headers = {}
```

Comment:
Maybe `default_headers` would be less confusing?

Reply:
Done :)